apps/web/lib/integrations/calendar/interfaces/Calendar.ts

```diff
@@ -55,6 +55,7 @@ export interface CalendarEvent {
   destinationCalendar?: DestinationCalendar | null;
   cancellationReason?: string | null;
   rejectionReason?: string | null;
+  disableNotes?: boolean;
```

Comment:
Let's optimize for legibility. `disableNotes` is way too open for interpretation.

```suggestion
  hideCalendarNotes?: boolean;
```

or 


```suggestion
  hideNotesFromCalendar?: boolean;
```